express-api/src/services/notifications/notificationServices.ts

```diff
@@ -163,11 +169,42 @@ const generateProjectNotifications = async (project: Project, previousStatusId:
         insertProjectNotificationQueue(template, projStatusNotif, project, project.Agency),
       );
     } else if (template.Audience == NotificationAudience.Agencies) {
-      //Will eventually integrate agency interest response lookup here.
+      const agencies = await AppDataSource.getRepository(Agency).find({
+        where: {
+          Id: In(
+            project.AgencyResponses.filter(
+              (resp) => resp.Response == AgencyResponseType.Subscribe,
+            ).map((a) => a.AgencyId),
+          ),
+          IsDisabled: false,
+          SendEmail: true,
+        },
+      });
+      agencies.forEach((agc) =>
+        returnNotifications.push(
+          insertProjectNotificationQueue(template, projStatusNotif, project, agc),
+        ),
+      );
     } else if (template.Audience == NotificationAudience.ParentAgencies) {
-      //Same here
+      const agencies = await AppDataSource.getRepository(Agency).find({
+        where: {
+          Id: In(
+            project.AgencyResponses.filter(
+              (resp) => resp.Response == AgencyResponseType.Subscribe,
+            ).map((a) => a.AgencyId),
+          ),
+          IsDisabled: false,
+          SendEmail: true,
+          ParentId: IsNull(),
+        },
+      });
+      agencies.forEach((agc) =>
+        returnNotifications.push(
+          insertProjectNotificationQueue(template, projStatusNotif, project, agc),
+        ),
+      );
     } else if (template.Audience == NotificationAudience.WatchingAgencies) {
-      //Same here
+      // Do we need this distinction?
```

Comment:
What is a watching agency if not a Subscribed agency?

Reply:
I don't know. I've been talking to Josh about it a bit and the distinction is pretty slim in the old PIMS implementation. The only real difference between what gets sent between Subscribed and Watch is that Subscribed gets an extra email right as the project enters Approved for ERP, otherwise they both just get sent emails after 30, 60, 90 days. The other weird thing is that with the way everything is set up, it looks like an agency on Subscribed needs to be a parent agency to get any notifications, but if they are on Interested it doesn't matter.